### ISSUE TITLE
[🐛 Bug]: Selenium 4 automation on Edge in compatibility mode with IE can not use dropdown objects

### ISSUE THREAD

**Issue by MajinKLS** [SeleniumHQ/selenium#11562]
### What happened?

Hi, I'm new to this community. Thanks to all of you who are making this project possible.

I'm working to migrate our Selenium 3 grid to Selenium 4 in the same operation mode, hub - node.

Running an automation of a private web application I found that a pair of webelements formed by a dropdown class and his dropdown-content class seems to do nothing when Selenium 4 try to click on the parent one. The dropdown object was found, got focused and apparently got a click event, but the dropdown-content class does not change his visibility.

This was tested in Selenium 3 and works fine. With Selenium 4 the issue appears using the compatibility mode of Edge with Internet Explorer. Other browsers like Chrome and Edge works fine.

### How can we reproduce the issue?

```shell
Unfortunately, I can't share or give access to the private application, but our application works mostly like the W3C explanation example:

https://www.w3schools.com/css/tryit.asp?filename=trycss_dropdown_text

With this app I was not able to reproduce the issue, so I made the following approach... the only difference I found in our application is that it manages click events with a javascript function nested to jQuery instead of listening to mouse hover over CSS like in the example. Something like:


$('cssSelectorChain > .dropdown > h > span').click(function() {
    /*code to get parent object and the get the dropdown-content*/
    var dropC = ...().find('.dropdown-content');
    var visibility = $(dropC).css('display');
   /* some conmutation of display property between values and application of the new value */
    ...
    ...
   $(dropC).css('display', visibility);
});
 
 I guess the issue must be related to this implementation.
```

### Relevant log output

```shell
I did not found any relevant log as the automation does not throw any exception when the click is made over the dropdown class.

Feel free to ask me anything and I'll try to answer you.
```

### Operating System

Windows 10 and Windows server 2016

### Selenium version

Java 4.7.2 running with Java 8 and Java 11

### What are the browser(s) and version(s) where you see this issue?

Edge compatibility with Internet Explorer, versions 109 and down

### What are the browser driver(s) and version(s) where you see this issue?

IEDriverServer 32b 4.7.0.0

### Are you using Selenium Grid?

Yes

**Comment by MajinKLS** [comment]
Hi,
I was working to reproduce the issue in a shareable example, and it seems I succeed. Doing this, I suspected what the problem is and seems that I solved it in my local workstation, but now, the problem is that I'm not able to get it back. So, I'm not sure at all what is the source of the issue, but I'm suspecting is not related to Selenium 4.

Now it seems more like some sort of configuration problem on script and Active X execution properties of Internet Explorer. I need to test in other workstations that currently are reproducing the issue and try to isolate it.

If I get some answer, I'll share to you.
